<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapPropertiesDialog</class>
 <widget class="QWidget" name="MapPropertiesDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>YaRXP - Map Properties</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="3" column="0">
      <widget class="QLineEdit" name="line_bgs">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="line_bgm">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QToolButton" name="button_bgm">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QToolButton" name="button_bgs">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QCheckBox" name="check_auto_change_bgs">
       <property name="text">
        <string>Auto-Change BGS</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QCheckBox" name="check_auto_change_bgm">
       <property name="text">
        <string>Auto-Change BGM</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Tileset:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="3">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Dimensions:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spin_x">
         <property name="minimum">
          <number>20</number>
         </property>
         <property name="maximum">
          <number>500</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spin_y">
         <property name="minimum">
          <number>15</number>
         </property>
         <property name="maximum">
          <number>500</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Map ID:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLineEdit" name="line_id">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLineEdit" name="line_name"/>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QComboBox" name="combo_tileset"/>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="button_ok">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1" rowspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Encounters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Steps Average</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QTableWidget" name="table_encounters">
        <column>
         <property name="text">
          <string>Troops</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>troop_id</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="spin_steps">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>999</number>
        </property>
        <property name="value">
         <number>30</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="button_add">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="button_del">
          <property name="text">
           <string>Del</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>line_id</tabstop>
  <tabstop>line_name</tabstop>
  <tabstop>combo_tileset</tabstop>
  <tabstop>spin_x</tabstop>
  <tabstop>spin_y</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>check_auto_change_bgm</sender>
   <signal>toggled(bool)</signal>
   <receiver>line_bgm</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>22</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>67</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>check_auto_change_bgm</sender>
   <signal>toggled(bool)</signal>
   <receiver>button_bgm</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>check_auto_change_bgs</sender>
   <signal>toggled(bool)</signal>
   <receiver>line_bgs</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>39</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>65</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>check_auto_change_bgs</sender>
   <signal>toggled(bool)</signal>
   <receiver>button_bgs</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>221</x>
     <y>246</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
